src/main/clojure/com/github/ericdallo/clj4intellij/configuration.clj

```diff
@@ -0,0 +1,13 @@
+(ns com.github.ericdallo.clj4intellij.configuration
+  (:import
+   [com.intellij.execution ProgramRunnerUtil]
+   [com.intellij.execution.executors DefaultRunExecutor]))
+
+(defn execute-configuration
```

Comment:
hum, this looks like more a util for clojure-repl-intellij, and only for tests too

Reply:
actually is not only for tests, is how you can run a Configuration programmatically.
Any plugin that uses Configuration can use. I can keep on clojure-repl-intellij, but I think that is generic enough to be here.

Reply:
[ref](https://plugins.jetbrains.com/docs/intellij/run-configurations.html#starting-a-run-configuration-programmatically)

Reply:
Yeah I know, but this is so generic that I don't see much value creating clojure API for that, is like create clojure function APIs for every Editor, Project and other services, we are not doing much on this function

Reply:
make sense